lib/python/vdm/main.py

```diff
@@ -69,4 +70,7 @@
 except (IOError, OSError, ImportError), e:
     sys.stderr.write('Exception (%s): %s\n' % (e.__class__.__name__, str(e)))
```

Comment:
why are we not conveying the stack trace? this applies to all exception reporting